scripts/setup.py

```diff
@@ -0,0 +1,57 @@
+import shutil
+import platform
+import subprocess
+import os
+
+OS_NAME = platform.system()
+
+# INSTRUCTIONS:
+# This script will move the pre-commit script from scripts folder to .git/hooks folder
+# to activate the checks prior to any commits.
+#
+# Run the script from the oppia-android root folder:
+#
+#   python scripts/setup.py
+#
+
+def install_hook():
+    """Installs the pre_commit_hook script and makes it executable.
+    It ensures that oppia-android/ is the root folder.
+    """
+    oppia_android_dir = os.getcwd()
+    hooks_dir = os.path.join(oppia_android_dir, '.git', 'hooks')
+    pre_commit_file = os.path.join(hooks_dir, 'pre-commit')
+    chmod_cmd = ['chmod', '+x', pre_commit_file]
+    file_exists = os.path.exists(pre_commit_file)
+    if file_exists:
+        print('Pre-commit hook already exists')
```

Comment:
The downside to this is that, if we make any changes to the pre-commit hook, it will not get updated to folks who already have the hook installed. Maybe we should not do this and just replace the file even if it exists (Since this script shouldn't be run very often anyway)

Reply:
now with bash, what we are doing is moving of file.